enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/ChartTypeSelectorList/ChartTypeDropdown.tsx

```diff
@@ -67,6 +67,7 @@ export const ChartTypeDropdown = () => {
           px={undefined}
           pr="md"
           rightIcon={<Icon ml="xs" size={10} name="chevrondown" />}
+          bg="var(--mb-color-bg-sdk-chart-type-selector)"
```

Comment:
All of these CSS variables has good fallback defined in `default-component-theme`, fyi